src/components/Breadcrumbs.tsx

```diff
@@ -79,22 +78,29 @@ const Breadcrumbs = ({
           sx={{ color: theme.palette.primary.contrastText }}
           {...muiBreadcrumbsProps}
         >
-          <Link key={"crumb-0"} underline="hover" color="inherit" href="/">
+          <MuiLink
+            key={"crumb-0"}
+            underline="hover"
+            color="inherit"
+            component={Link}
```

Comment:
I did wonder whether this should be NavLink rather than Link but I think this is right for breadcrumbs.

Reply:
Agreed.